_posts/2023-05-03-patternfly-v5-alpha.md

```diff
@@ -2,7 +2,7 @@
 
 PatternFly team has been working with “all hands on deck” on the version 5. That new major release is still scheduled for June. They have conducted two rounds of alpha testing so far and they've gone pretty well. Meanwhile PatternFly 4 has been in development freeze for a while as the team is focusing on the new version.
 
-**Cockpit has decided to go ahead with an early adoption of PatternFly 5. **The motivation for adopting PatternFly 5 can be summarized in several key points. Firstly, the early adoption of PatternFly 5 is advantageous for the PatternFly team. Secondly, as PatternFly 5 is still in its alpha phase, it is more receptive to significant changes. Thirdly, the high code coverage of Cockpit, with its integration and image comparison testing, provides confidence in being one of the early adopters without compromising product quality. Finally, PatternFly 4 is in development freeze, which hinders the ability to utilize the latest features and fixes found in PatternFly 5.
+<b>Cockpit has decided to go ahead with an early adoption of PatternFly 5.</b>The motivation for adopting PatternFly 5 can be summarized in several key points. Firstly, the early adoption of PatternFly 5 is advantageous for the PatternFly team. Secondly, as PatternFly 5 is still in its alpha phase, it is more receptive to significant changes. Thirdly, the high code coverage of Cockpit, with its integration and image comparison testing, provides confidence in being one of the early adopters without compromising product quality. Finally, PatternFly 4 is in development freeze, which hinders the ability to utilize the latest features and fixes found in PatternFly 5.
```

Comment:
Please don't use HTML markup in Markdown. `**` was correct, but it lacked a space after the second `**`.